cmd/podman/root.go

```diff
@@ -1,6 +1,7 @@
 package main
 
 import (
+	goerrors "errors"
```

Comment:
As this point, I don't think we can swap over to Golang's `errors` package without doing it everywhere throughout the stack - there is a high chance the error we're working with came from `github.com/pkg/errors` and was wrapped. Recommend you just use `errors.Cause()` instead.